dev/integration_tests/display_cutout_rotation/.metadata

```diff
@@ -0,0 +1,30 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled and should not be manually edited.
+
+version:
+  revision: "0dc4eb31df6fe16c1bac10bef3904eb378056c35"
+  channel: "[user-branch]"
+
+project_type: app
+
+# Tracks metadata for the flutter migrate command
+migration:
+  platforms:
+    - platform: root
+      create_revision: 0dc4eb31df6fe16c1bac10bef3904eb378056c35
+      base_revision: 0dc4eb31df6fe16c1bac10bef3904eb378056c35
+    - platform: android
+      create_revision: 0dc4eb31df6fe16c1bac10bef3904eb378056c35
+      base_revision: 0dc4eb31df6fe16c1bac10bef3904eb378056c35
```

Comment:
Doesn't matter for this pr, but do you know where this `.metadata` file comes from? Should we remove the output for the `flutter migrate` command, given that it is probably not going to land at any point?

Reply:
I do not. I should talk to @jmagman about if we should pick the migration work back up.